MANIFEST.in

```diff
@@ -1,3 +1,3 @@
 include LICENSE
 include README.md
-recursive-include src/ashley *.html *.png *.gif *.js *.css *.jpg *.jpeg *.po *.mo *.eot *.svg *.ttf *.woff *.woff2
+recursive-include src/ashley upgrades *.html *.png *.gif *.js *.css *.jpg *.jpeg *.po *.mo *.eot *.svg *.ttf *.woff *.woff2
```

Comment:
I'm not sure how it can be used from a python package... :thinking:

Reply:
You're right, it's irrelevant. I removed it from the manifest and updated the Changelog and commit message.